answer_questions_using_vector_search_weaviate/README.md

```diff
@@ -0,0 +1,28 @@
+# Answer questions by performing similarity search on embedding vectors stored in Weaviate
+
+## Use case
+Ask questions from knowledge stored in a Weaviate cluster using embedding similarity search. Embedding vectors are retreived using OpenAI GPT3 models. 
```

Comment:
```suggestion
Ask questions from knowledge stored in a Weaviate cluster using embedding similarity search. Embedding vectors are retrieved using OpenAI GPT3 models. 
```

Reply:
Addressed in https://github.com/ballerina-guides/ai-samples/pull/60/commits/b266f95fb9a512010774915db533652910a42502.